src/ResourceManager/Automation/Commands.Automation.Test/ScenarioTests/Resources/Test.ps1

```diff
@@ -0,0 +1,3 @@
+get-date
```

Comment:
I assume these snippets are running in a runbook, is this correct?

Reply:
Yes that is correct